### PR TITLE
Fixed an error + unneeded lines + words

### PR DESCRIPTION
**Pull request by SamuelMarks**

### CODE REVIEW

File: sections/failing_vm.html [hunk 1]
```diff
@@ -6,11 +6,9 @@
 
 <div class="terminal-block">
   <h4 class="terminal-code-text">$ vagrant ssh</h4>
-  <h4 class="terminal-code-text">vagrant@agent-id-bosh-0$ sudo apt-get update</h4>
-  <h4 class="terminal-code-text">vagrant@agent-id-bosh-0$ sudo apt-get install curl</h4>
-  <h4 class="terminal-code-text">vagrant@agent-id-bosh-0$ curl http://127.0.0.1:7777/containers</h4>
+  <h4 class="terminal-code-text">vagrant@agent-id-bosh-0$ wget -qO - http://127.0.0.1:7777/containers</h4>
```
Comment (mariash) [inline sections/failing_vm.html:9]: nice!

File: sections/failing_service.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <%= circled_title(1, "Failing service") %>
 
-<p>BOSH is using <a href="https://mmonit.com/monit/">monit</a> to monitor running services. If the service goes down it will bring it up. Let's watch how this works. SSH to one of VMs (provide any temporary password when prompted, same password will be used for sudo):</p>
+<p>BOSH is using <a href="https://mmonit.com/monit/">monit</a> to monitor running services. If the service goes down it will bring it up. Let's watch how this works. SSH to one of VMs:</p>
```
Comment (mariash) [inline sections/failing_service.html:3]: good catch, we no longer need to provide passwords!

File: sections/failing_service.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
 <div class="terminal-block">
   <h4 class="terminal-code-text">$ bosh ssh app/0</h4>
-  <h4 class="terminal-code-text">$ sudo su</h4>
-  <h4 class="terminal-code-text">$ while monit summary; do sleep 1; done</h4>
+  <h4 class="terminal-code-text">$ sudo -i</h4>
```
Comment (mariash) [inline sections/failing_service.html:7]: interesting to know why `-i` is better? we don't need to simulate initial login..

**Comment by mariash** [comment]
Thanks for PR, nice improvements!

File: sections/failing_service.html (hunk 3 above)
Comment (SamuelMarks) [inline sections/failing_service.html:7]: http://askubuntu.com/questions/331062/what-is-the-functional-difference-between-sudo-su-and-sudo-i

**Comment by SamuelMarks** [comment]
:)

**Pull request merged**